test/test_flows.py

```diff
@@ -29,8 +43,11 @@ def test_flows(flow_class, flow_args, input_dim, batch_shape):
 
     # test inverse is correct
     y = transform(x)
-    inv = transform.inv(y)
-    assert_allclose(x, inv, atol=1e-5)
+    try:
+        inv = transform.inv(y)
+        assert_allclose(x, inv, atol=1e-5)
+    except RuntimeError:
```

Comment:
i guess this should be `NotImplementedError` ?

Reply:
Thanks, I have replaced `RuntimeError` by `NotImplementedError` in the implementation.